crates/telio-model/src/features.rs

```diff
@@ -535,6 +535,9 @@ pub struct Features {
     /// PMTU discovery configuration, enabled by default
     #[serde(default = "FeaturePmtuDiscovery::serde_default")]
     pub pmtu_discovery: Option<FeaturePmtuDiscovery>,
+    /// Multicast support
+    #[serde(default)]
+    pub multicast: bool,
```

Comment:
Since this is part of the boundary with external world, I think there should be 4 types of tests:
- what happens when the field is missing
- what happens when the field is set to true
- what happens when the field is set to false (afaics, this is already present)
- what happens when the field is set to incorrect value

Reply:
>what happens when the field is set to incorrect value
That would be nice, but do we have any tests of this kind?

Reply:
When I added it I was mostly looking at what tests were there already for the features struct and didn't think more of it, but you are right, we should be better about testing this. I added more tests to this section, and hopefully we can add similar tests for all the other fields in the `Features` struct in the near future